### ISSUE TITLE
Overleaf template

### ISSUE THREAD

**Issue by Thuener** [schardong/thesisPUC-Template#3]
Now everyone is using overleaf. I think it is a good idea to put these 4 examples (pt_msc, pt_phd, en_msc, and en_phd) as a template in overleaf. What do you think?
https://www.overleaf.com/learn/how-to/Can_I_create_my_own_LaTeX_templates%3F

**Comment by schardong** [comment]
I tried creating an overleaf template some time ago, however there is one already by another author (https://www.overleaf.com/latex/templates/puc-rio-thesis-and-dissertation-template/dbbybvkrrntm). Unfortunately that template does not incorporate some of the corrections we made.

**Comment by augustoicaro** [comment]
Hello, @schardong  and @Thuener I'm the person that updates the overleaf version on professor Marcelo Gattass behalf. Will be great to incorporate your changes to the overleaf template. If you're interested in that we can discuss how to proceed.

**Comment by schardong** [comment]
Hello @augustoicaro. That would be great! The changes we made here are basically some ad-hoc fixes to issues found during the certification process. I made this repo to facilitate their distribution. It would be nice to have them incorporated in a more official repository!

**Comment by augustoicaro** [comment]
Great! I will check your commits and start to add the changes in my code. But the distribution problem still persist. My overleaf version is different from the package provided in PUC-Rio website. The main problem now is that professor Casanova is responsable to aprrove changes to the website. I tried for almost one year unify the model, but no success.

I will try again next year. And find a instituonal place to put the code, so more people can contribute easily.

I also talk with the original maintainer, David, but he is a GNU centered person and disagree with the use of mainstream tools that is not open source. The "official" git repo is in sorcehut.

**Comment by schardong** [comment]
OK! One of the aspects that I did not fix yet is the keywords and abstract keywords. One of them must end in colons, while the other is used for the catalographic file. I created a new command to define them, but they must by duplicated, as in:

```latex
\keywords
{
  \key{Redução de Cenários}
  \key{Interatividade}
  \key{Visualização Científica}
  \key{Tomada de Decisão}
  \key{Conjuntos de Dados Temporais}
}

\keywordsabstract
{
  \key{Redução de Cenários;}
  \key{Interatividade;}
  \key{Visualização Científica;}
  \key{Tomada de Decisão;}
  \key{Conjuntos de Dados Temporais.}
}
```

I'm not particularly proud of this, but it worked at the time. I've had little time to fix this, unfortunately.

As for the distribution, maybe the official source tree just needs more visibility. I didn't know that there was a git controlled source for the template, honestly. If a link was added to the CCPG website, it would help.

**Comment by augustoicaro** [comment]
Yeah! I agree with you about your visibility suggestion. I will talk directly with professor Casanova and try to organize the source control and the CCPG website to reflect that. And make it easier for the people that want to contribute. The problem with the original git is that they use a niche-style organization following the GNU principles. They don't use mainstream services, and in source hunt, you need to make your commits using git mail. I tried a few commits, but it's not for everybody.
